src/commands/publish.rs

```diff
@@ -42,82 +49,104 @@ impl PublishCommand {
     }
 }
 
-/// Start a new package publish.
+/// Publish a package to a warg registry.
 #[derive(Args)]
-pub struct PublishStartCommand {
+#[clap(disable_version_flag = true)]
+pub struct PublishReleaseCommand {
     /// The common command options.
     #[clap(flatten)]
     pub common: CommonOptions,
     /// Use to initialize a new package.
     #[clap(long)]
     pub init: bool,
+    /// Queue the release for publishing with the `publish submit` command.
+    #[clap(long)]
+    pub queue: bool,
```

Comment:
The proposed `--queue` flag would have to be added to every future command that can be deferred (e.g. yank, grant, revoke). I'm not sure I like the ergonomics of this.

Reply:
After testing the DX a bit, I decided to just put back the `publish start` command, but also add a `publish init` command that either enqueues an init record (for a pending publish) or immediately tries to initialize the package if there is no pending publish, similar to how `publish release` now operates.